### DISCUSSION TITLE
I can't see a pyright error that goes beyond the edge of the screen

### DISCUSSION THREAD

**Discussion by ghost** [NvChad/NvChad#950] [Q&A] [answered]
[My dotfiles](https://codeberg.org/LifeSymbiont/dotfiles/src/branch/main/config/nvim/lua/custom). I'm using pyright and when there are errors I can only see part of them because they go beyond the edge of the screen. How can I see the complete error message?

**Comment by amirhhashemi** [comment, marked as answer]
`<leader>ge` will show the diagnostics of the current line in a floating window so you should be able to see the whole error.

Personally, I disabled the virtual text and only use the above mapping to see the diagnostics.
